packages/requests/src/lib/requests-cache.spec.ts

```diff
@@ -91,4 +92,21 @@ describe('requestsCache', () => {
     subject.next({});
     expect(spy).toHaveBeenCalledTimes(0);
   });
+
+  it('should uphold ttl', () => {
+    store.reduce(
+      updateRequestCache(requestKey, 'full', 1000)
+    );
+
+    expect(
+      store.query(isRequestCached(requestKey, { value: 'full' }))
+    ).toBeTruthy();
+
+    setTimeout(() => {
+      console.log("Time's up -- stop!");
+      expect(
```

Comment:
I'm pretty sure this is a false positive.

Reply:
increased time difference, tests seems to always pass. why false positive?

Reply:
Try to fail it.

Reply:
You need to use jest fake timers.